ui/components/app/wallet-overview/act-overview.js

```diff
@@ -129,7 +129,7 @@ const ActOverview = () => {
           {t('send')}
         </button>
       </div>
-    </div>
+    </div >
```

Comment:
```suggestion
    </div>
```
- Please run `yarn lint:fix`